### ISSUE TITLE
Cannot Launch Accessibility Insights on E2E Fabric Test App

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#11594]
### Problem Description

After pulling in the latest changes from main, launching Accessibility Insights on the E2E test app on Fabric results in an alert window being thrown. The alert windows states that an assert has been hit in the ScrollViewComponentView.cpp file (https://github.com/microsoft/react-native-windows/blob/d95f6c8632a8a6421d2a322e293caec01d66dc7d/vnext/Microsoft.ReactNative/Fabric/Composition/ScrollViewComponentView.cpp#L222). 

Have not yet tested if this occurs for the playground app, but issue did not occur previously. The assert in line 222 has been present in the files since ScrollView has been present since the start of this file on Fabric, so we'll have to figure out why it is now being hit when Accessibility Insights is launched. 

### Steps To Reproduce

Run e2e-test-app-fabric. 
Open Accessibility Insights.

### Expected Results

No errors.

### CLI version

npx react-native -v

### Environment

```markdown
npx react-native info
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by chrisglein** [comment]
@chiaramooney Can you provide the callstack to know how it's being called?
This method isn't yet implemented.
We should make sure that the accessibility code isn't expecting view props from things that aren't view props.
Other than that, the fix would be to implement this.

Certainly this will block https://github.com/microsoft/react-native-windows/issues/11296
Not tracked in https://github.com/microsoft/react-native-windows/issues/11152 because it's not an end developer facing method.

**Comment by chiaramooney** [comment]
Method is being called in CompositionDynamicAutomationProvider.cpp at line 210. It seems to be seeking...
```cpp
 auto props = std::static_pointer_cast<const facebook::react::ViewProps>(strongView->props());
  if (props == nullptr)
    return UIA_E_ELEMENTNOTAVAILABLE;
```
@acoates-ms for fyi.
